### ISSUE TITLE
task.yml should include BOSH_ALL_PROXY

### ISSUE THREAD

**Issue by rob-greene** [pivotal-cf/bbr-pcf-pipeline-tasks#26]
We initiate CF backups via a bastion host and are starting to see `[WARNING] BOSH_ALL_PROXY was defined in pipeline but missing from task file`. 

As of Concourse 4.1 this is a warning but the 4.1 release notes indicate this will become an error in the future.

**Comment by glestaris** [comment]
Hey @rob-greene, we converted this into a feature request and prioritsed it in our backlog. We will get back to you as soon as we add this.

**Comment by rob-greene** [comment]
Thanks! We can't be the only ones working through the proxy. ;-)

**Comment by MirahImage** [comment]
Hi @rob-greene, 

Thanks for bringing this to our attention. We've updated the `bbr-backup-ert` task to take `OPSMAN_PRIVATE_KEY` as an optional parameter. If the parameter is set, the task will run `bbr deployment ...` using `BOSH_ALL_PROXY`. 

Note that `bbr director ...` does not currently support `BOSH_ALL_PROXY`, so we have not changed the `bbr-backup-director` task. 

We have also updated the README to reflect this. 

Best,
Mirah and @jamesjoshuahill
